packages/glimmer-test-helpers/lib/environment.ts

```diff
@@ -520,8 +525,12 @@ export class TestEnvironment extends Environment {
     return this.components[<string>name[0]];
   }
 
-  compile(template: string) {
-    return rawCompile(template, { env: this });
+  compile(template: string, options?: CompileOptions) {
+    let rawOptions = { env: this, moduleName: null };
+    if (options) {
```

Comment:
Since this compile function is specific to the test environment, we need to do this guarding in order to provide a default moduleName. moduleName is not relevant to a lot of tests.